client/src/css/structure.css

```diff
@@ -888,19 +888,35 @@ a[disabled] {
   background-color: #ff000083 !important;
 }
 
+.icon-expired {
+  color: #ff000083 !important;
```

Comment:
I encourage you to just use the 6-digit version here.  You do not need or want any transparency in the legend!

Reply:
I agree with this suggestion.